Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.yml

```diff
@@ -1841,6 +1851,15 @@ script:
     - name: entity_id
       description: Entity id.
       required: false
+  - name: checkpointhec-download-email
+    description: Download email file.
+    arguments:
+    - name: entity
+      description: Email entity id.
+      required: true
+    - name: original
```

Comment:
Let's rewrite the description here,and replace the argument with predefined argument, example "whether to download the original email or not"

Suggested change:

      required: false  
      auto: PREDEFINED  
      predefined:  
      - True  
      - False